docs/userguide/examples/run_increment.py

```diff
@@ -0,0 +1,10 @@
+from parsl.dataflow.dflow import DataFlowKernelLoader
+
+from config import local_threads
+DataFlowKernelLoader.load(local_threads)
```

Comment:
> mandatory init operation.

It's not mandatory (you can still pass the DFK to the app decorator instead), but I agree it's long. I like your suggestion and will implement it. One thing to consider is that you *can* pass kwargs which are passed on to the DFK, so you'd need to look at the DFK docs to understand what is possible there. I am hoping when we re-work the config we can eliminate most of that, so that the main interface the user needs to deal with is the `Config` class. So for now I'm fine with getting rid of `DataFlowKernel` in the name here but still saying "these kwargs will be passed to the DFK", with the understanding that when the configs are tweaked we should be able to remove the reference to the DFK and just say "pass a `Config`", which the user can read about in the `Config` docs)

Reply:
On second thought, to keep things simple, I'll just ditch passing the kwargs, and assume the user will manipulate the config dict beforehand if they need to.